activesupport/lib/active_support/claims.rb

```diff
@@ -0,0 +1,57 @@
+require 'active_support/time'
+
+module ActiveSupport
+  class Claims
```

Comment:
Yes, but that’s defined under `private` right? Because I believe stuff under there doesn’t show up in the documentation. But I don’t know if that’s true for classes. Eh, let’s add one to be safe. Then people can always see it’s not needed :smile:

> Den 10/07/2015 kl. 22.02 skrev Siddharth Bhatore <notifications@github.com>:
> 
> In activesupport/lib/active_support/claims.rb <https://github.com/sbhatore/rails/pull/3#discussion_r34390948>:
> 
> > @@ -0,0 +1,57 @@
> > +require 'active_support/time'
> > +
> > +module ActiveSupport
> > +  class Claims
> Also, we would not want people to have a look at LegacyMessageVerifier either, right ? So, a :nodoc: for that too would be needed.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/sbhatore/rails/pull/3/files#r34390948>.
> 

--
Kasper